website/docs/r/action_group.html.markdown

```diff
@@ -0,0 +1,99 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_action_group"
+sidebar_current: "docs-azurerm-resource-action-group"
+description: |-
+  Manages an Action Group of Azure monitoring service
+
+---
+
+# azurerm_action_group
+
+Manages an Action Group of Azure monitoring service.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "test" {
+  name     = "monitoring-resources"
+  location = "West US"
+}
+
+resource "azurerm_action_group" "test" {
+  name                = "CriticalAlertsAction"
+  location            = "Global"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  short_name          = "p0action"
+
+  email_receiver {
+    name          = "sendtoadmin"
+    email_address = "admin@contoso.com"
+  }
+
+  email_receiver {
+    name          = "sendtodevops"
+    email_address = "devops@contoso.com"
+  }
+
+  sms_receiver {
+    name         = "oncallmsg"
+    country_code = "1"
+    phone_number = "1231231234"
+  }
+
+  webhook_receiver {
+    name        = "callmyapiaswell"
+    service_uri = "http://example.com/alert"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the Action Group. Changing this forces a new resource to be created.
+* `location` - (Required) The location of this Action Group. The only possible value is `Global`.
```

Comment:
if the only possible value is `Global` then this field should be hard-coded and shouldn't be available for users to set